en/components/grid/selection_based_aggregates.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Angular Selection-based Data Aggregation | selection | Ignite UI for Angular | Infragistics 
+_description: Learn how to aggregate selected data in a grid with the Ignite Angular Material table using rich API. It also allows instant content aggregations in the virtualized data of the Grid
+_keywords: Data aggregation, selection, ignite ui for angular, infragistics
```

Comment:
Shouldn't there be keywords like "grid", "table", etc?

Reply:
According to the SEO guidance document, the `Meta keywords` should contain the primary keyword, the name of the product and company name. If the primary keyword is "Data aggregation", the 'Meta keywords' seem to be correct.